package.json

```diff
@@ -32,7 +32,7 @@
     "dev:browser": "vite",
     "dev:electron": "electron-vite -w",
     "build": "npm run build:browser && npm run build:electron",
-    "build:browser": "vite build",
+    "build:browser": "rm -rf plugins/dist-browser && vite build && mv dist-browser plugins/",
```

Comment:
Can you define a separate command for plugin use? Or 1) move the `rm` part into `clean:browser` command and call `cp` instead of `mv` to keep the original `dist-browser` for non-plugin usage

Reply:
@KazuCocoa Fixed!